assets/styles/layout/_forms.scss

```diff
@@ -525,3 +525,36 @@ div.input-box {
     border-radius: var(--kbin-rounded-edges-radius) !important;
   }
 }
+
+.related-link-fieldset {
+    border: 0;
+    margin: 0;
+    padding: 0;
+}
+
+.related-link-fieldset legend {
+  margin-bottom: 0.5rem;
+}
+
+.related-link-row {
+    display: flex;
+    flex-wrap: nowrap;
+    row-gap: 0.5rem;
+    column-gap: 0.2rem;
+    align-items: center;
+    margin-bottom: 0.25rem;
+}
+
+.related-link-row div {
+  margin-bottom: 0;
+  flex-grow: 1
+}
+
+.related-link-row button {
+    padding: 0;
+    background: none;
+    font-size: 1.5rem;
+    border: 0;
+    color: var(--kbin-input-text-color);
+
+}
```

Comment:
please remove that and add the class `btn__secondary` to the trash icon buttons. Then they will look like other buttons in our UI

Reply:
modified